### PR TITLE
Fix serialization. Donation & Request relationship.

### PR DESCRIPTION
**Pull request by Everod52**
Added relationship to Donation with request

### CODE REVIEW

**Review by yetsielaviles — APPROVED** [review]

**Review by jaortiz117 — APPROVED** [review]

**Review by jaortiz117 — CHANGES_REQUESTED** [review]
Voy a hacer los cambios al donations dao

**Comment by Everod52** [comment]
> Voy a hacer los cambios al donations dao

No hay problema.

File: dao/mixin.py [hunk 1]
```diff
@@ -20,16 +20,17 @@ def to_dict(self, rel=None, backref=None):
...
+                # if backref and backref == relation.table:
+                #     continue
+                # value = getattr(self, attr)
+                # if value is None:
+                #     res[relation.key] = None
+                # elif isinstance(value.__class__, DeclarativeMeta):
+                #     res[relation.key] = value.to_dict(backref=self.__table__)
+                # else:
+                #     res[relation.key] = [i.to_dict(backref=self.__table__)
+                #                          for i in value]
```
Comment (Everod52) [inline dao/mixin.py]: Delete commented code and TODO

File: dao/request.py [hunk 2]
```diff
@@ -1,15 +1,19 @@
 import datetime
 
 from config import db
+from dao.mixin import OutputMixin
 
 
-class Request(db.Model):
+class Request(OutputMixin, db.Model):
+    RELATIONSHIPS_TO_DICT = True
+
```
Comment (Everod52) [inline dao/request.py:10]: Add REQUEST_REQUIRED_PARAMS list

**Review by Ale573 — CHANGES_REQUESTED** [review]
When you add a donation to the system, the gets doesn't work for the user. There is a problem with the dictionary.

**Review by Ale573 — APPROVED** [review]
Everything is working. All the gets, posts, updates, and deletes for users and donations.

**Review by javihernovoa — APPROVED** [review]

**Pull request merged**